api/config/project/volumes/calloutImages--f52f3e9c-434e-43b5-89eb-a5776e6bc4a6.yaml

```diff
@@ -41,7 +41,14 @@ handle: calloutImages
 hasUrls: true
 name: 'Callout Images'
 settings:
-  path: '@webroot/assets/callout'
+  bucket: rubin-obs-api_assets_callouts
```

Comment:
Should these be environment variables/Craft aliases instead? I hardcoded the names of the buckets on the fly

Reply:
yes, good idea.  I'll update.

Reply:
Thank you sir